todo.ts

```diff
@@ -10,8 +10,7 @@ export interface Todo {
 /**
  * parseTodoID parses the ID of a todo from a URL.
  */
-export function parseTodoID(url: string): string {
-  const { 2: id } = url.split("/");
-  console.log({ url, id });
+export function parseTodoID(pathname: string): string {
+  const { 2: id } = pathname.split("/");
```

Comment:
maybe a bit clever with the de-structuring here?
I feel like i had to work a bit harder than necessary to figure out you're asking for the second element in the array/split string.

Reply:
I hope this is more clear: <https://github.com/acmcsufoss/api-workshop/pull/2/commits/e922f5c1ee58d9eaa00529f7830316ea808dda30>.

Reply:
Alternatively, I could keep the `const { 2: id } = pathname.split("/");` but with a new comment:
> Extracts the value at index 2 from the array returned by splitting the 'pathname' string using "/" as the delimiter and assigns it to the variable 'id'.
>
> See:
> https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment